tests/unit/services/resolvers/bonfida-resolver.service.spec.ts

```diff
@@ -46,4 +46,9 @@ describe('bonfida-resolver.service', () => {
 
         expect(bonfidaResolverService.resolve("nick.sol")).rejects.toThrow("Bonfida Error: Invalid name account provided");
     });
+
+    test('SHOULD do not get addresses for domain with network IF domain ends without .sol', async () => {
```

Comment:
Minor: SHOULD NOT

Also need to add "SHOULD get addresses for domain with network IF domain ends with .sol"